app/controllers/donations_controller.rb

```diff
@@ -64,9 +64,9 @@ def create_session
 
         SubscriptionMailer.with(subscription: subscription, email: email).edit.deliver_later
 
-        flash[:notice] = "We sent you an email with a link to do the thing you need to do."
+        flash[:notice] = 'We sent you an email with a link to do the thing you need to do.'
       else
-        flash[:error] = "We found multiple subscriptions with that email address."
+        flash[:error] = 'We found multiple subscriptions with that email address.'
```

Comment:
Can you move both of these strings into `en.yml` please?

Reply:
Totally. I left all text out of `en.yml` for now because everything still feels a little TBD. My plan was to wait until things are exactly where/how we want them and _then_ add everything there.